assets/pages/bundles-landing/helpers/subscriptionsLinks.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+// ----- Types ----- //
+
+type Product = 'paper' | 'digital' | 'paperDig';
+
+type PromoCodes = {
+  [Product]: string,
+};
+
+export type SubsUrls = {
+  [Product]: string,
+};
+
+
+// ----- Setup ----- //
+
+const subsUrl = 'https://subscribe.theguardian.com';
+
+const defaultPromos: PromoCodes = {
+  digital: 'p/DXX83X',
+  paper: 'p/GXX83P',
+  paperDig: 'p/GXX83X',
+};
+
+const tests = [
+  {
+    name: 'rebaseline',
+    intCmps: [
+      'gdnwb_copts_memco_sandc_epic_rebaseline_support_proposition_support_proposition',
+    ],
+    promoCodes: {
+      digital: 'p/DP83X',
+      paper: 'p/VHD83P',
+      paperDig: 'p/VHD83X',
+    },
+  },
+];
+
+
+// ----- Functions ----- //
+
+function buildUrls(promoCodes: PromoCodes, intCmp: string): SubsUrls {
+
+  const params = new URLSearchParams();
+  params.append('INTCMP', intCmp);
```

Comment:
Only other place I can think of is [here](https://github.com/guardian/support-frontend/blob/f1b2aef89c7e781d06eab3fec1b3f913e5a12ab1/assets/pages/bundles-landing/components/Bundles.jsx#L170), but it's doing other stuff too at that point.